src/ch/imedias/rsccfx/view/SupporterAttributesDialog.java

```diff
@@ -116,8 +112,6 @@ private void layoutForm() {
     attributePane.add(addressFld, 1, 1);
     attributePane.add(portLbl, 0, 2);
     attributePane.add(portFld, 1, 2);
-    attributePane.add(pictureLbl, 0, 3);
```

Comment:
I forgot to mention that in the PR, @Kennox removed it since we decided not to implement the picture fields